src/app/crypto.service.ts

```diff
@@ -279,3 +303,22 @@ export class CryptoService {
     return ethAddressChecksum;
   }
 }
+
+/**
+ * We set the account according to the following derivation path scheme:
+ * m / purpose' / coin_type' / account' / change / address_index
+ * See for more details: https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#account
+ */
+function deriveKeys(
```

Comment:
Can we call these "subaccountKeys" rather than "derived" keys to avoid overloading/confusion? We can call this function generateSubaccountKeys() maybe.

Reply:
yeah, i agree it's a little confusing. Will update in the other PR 👍🏻